### PR TITLE
Added NextJS topic

### PR DESCRIPTION
**Pull request by joaovictornsv**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by damaneice — COMMENTED** [review]
👋🏾 @joaovictornsv   It looks like there are a couple of ci build errors. Here are a couple of suggested changes that I think we fix those errors.

File: topics/nextjs/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+aliases: next, next-js
+created_by: Vercel
+display_name: Next
+github_url: https://github.com/vercel/next.js
+logo: nextjs.png
+released: October 25, 2016
+short_description: Next.js is an open-source React front-end development web framework
```
[the remark this answers is not in the document]
Comment (joaovictornsv) [reply topics/nextjs/index.md]: Hi @damaneice 👋. I made the suggested corrections. Thanks you for help! 😄

File: topics/nextjs/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+github_url: https://github.com/vercel/next.js
+logo: nextjs.png
+released: October 25, 2016
+short_description: Next.js is an open source React front-end development web framework.
+topic: nextjs
+url: https://nextjs.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Next.js
+---
+
+Next.js is an opensource React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React-based web applications.
```
Comment (damaneice) [inline topics/nextjs/index.md:14]:
```suggestion
Next.js is an open source React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React-based web applications.
```

**Review by damaneice — COMMENTED** [review]
Do you mind making one more small change?

File: topics/nextjs/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+github_url: https://github.com/vercel/next.js
+logo: nextjs.png
+released: October 25, 2016
+short_description: Next.js is an open-source React front-end development web framework
+topic: nextjs
+url: https://nextjs.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Next.js
+---
+
+Next.js is an open-source React front-end development web framework that enables functionality such as server-side rendering and generating static websites for React based web applications.
```
[the remark this answers is not in the document]
Comment (joaovictornsv) [reply topics/nextjs/index.md]: I forgot that little mistake 😅. Adjusted ✔

**Review by damaneice — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**